src/color.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* stylelint-disable custom-property-empty-line-before, declaration-no-important */
+
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host {
+  background-color: var(--ds-auro-menuoption-background-color);
+  color: var(--ds-auro-menuoption-text-color);
+}
+
+:host([disabled]) {
+  --ds-auro-menuoption-text-color: var(--ds-color-text-ui-disabled-default, $ds-color-text-ui-disabled-default);
+}
+
+:host(:hover:not([disabled])),
```

Comment:
Is this selector better handled by actually not using it and just adding `pointer-events: none;` to the `[disabled]` state?

Reply:
Im not sure what you are referring to, this rule adjusts the color.